mesonbuild/cmake/interpreter.py

```diff
@@ -67,6 +67,14 @@
     'CMP0102',
 ]
 
+# CMAKE is a bit more averse to debugging, but in spirit the build types match
+buildtype_map = {
+    'debug': 'Debug',
+    'debugoptimized': 'RelWithDebInfo',  # CMAKE sets NDEBUG
```

Comment:
```suggestion
    'debugoptimized': 'RelWithDebInfo',  # CMake sets NDEBUG
```

and the latter case too, `CMAKE` -> `CMake`